combat-trainer.lic

```diff
@@ -1853,12 +1870,14 @@ class ManipulateProcess
     @manip_to_train = settings.manipulate_only_to_train
     echo("  @manip_to_train: #{@manip_to_train}") if $debug_mode_ct
 
+    Flags.add('ct-manip-broke', 'You feel your link with .* suddenly break', 'You feel your link suddenly break', 'You feel your link with .* weakening with distance.')
```

Comment:
These returns could probably be condensed into a regex. `/^You feel your link (with)?.* (suddenly|weakening)?/` something like that. I'm not the best regexer.

https://rubular.com/r/rDPAFupLyToSJl

Reply:
Will work on it

Reply:
done.